### PR TITLE
Adding an ADR for GH labels decluttering (issue #1496 part 1)

### PR DESCRIPTION
**Pull request by iMichaela**
# Committer Notes

Adding an ADR that identifies the Git Hub labels we need and/or want to use and the ones we want to remove. The labels are also grouped per categories in this ADR.

For additional actions related to issue #1496 , a.k.a [Draft ADR How to Handle EPICs and Related Issues](https://hackmd.io/eUhzIQx3RW6zYcGZ_iIJUQ), additional internal meetings are necessary.
Minutes of the https://github.com/usnistgov/OSCAL/issues/1496 are available [here](Overall notes: https://hackmd.io/YS5ib1yMQ4ab-7Ocyy2vUA)
The ADR draft with comments is captured [here](https://hackmd.io/UrSjUKGiQRuiA2VJLG-Mpg)

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- ~Have you written new tests for your core changes, as applicable?~
- ~Have you included examples of how to use your new feature(s)?~
- ~Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.~

### CODE REVIEW

File: decisions/0003-github-labels-decluttering.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Use of Labels in OSCAL Project(s)
+
+Date: 02/17/2023
+
+## Status
+
+Feedback Requested
```
Comment (aj-stein-nist) [inline decisions/0003-github-labels-decluttering.md:7]: Non-blocking: I will not make this mistake again like I did on my own PR for the previou ADRs, once we are ready to merge this we switch it as needed.

```suggestion
Approved
```

File: decisions/0003-github-labels-decluttering.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+## Decision
+
+Existing labels were reviewed, a subset of labels is kept and new ones are defined.
+
+The lists of labels that follow will use across OSCAL repositories.
+
+#### General labels:
+
+- breaking change
```
Comment (aj-stein-nist) [inline decisions/0003-github-labels-decluttering.md:23]: This is a great start, thank you. As of the last few days, there was one I wanted to add that happened late after these conversations and I cannot recall immediately now. I will try to jump start my memory and update this with an additional recommendation once I recall. After that, I am ready to approve. (That, or I will need a follow-up PR like later today or the next day, seems silly on my part.)

Comment (iMichaela) [reply decisions/0003-github-labels-decluttering.md:23]: @aj-stein-nist I am ready to commit the label you want to add as soon as you remember it.

File: decisions/0003-github-labels-decluttering.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+- community feedback needed
+- developer experience
+- discussion needed
+- epic
```
Comment (aj-stein-nist) [inline decisions/0003-github-labels-decluttering.md:27]: Apologies, it was `faq` for FAQ, Frequently Asked Questions, I remember now. It was so we can use these labels in GitHub Discussions for the project and quickly filter on discussion posts that are candidates for FAQ status. https://github.com/usnistgov/OSCAL/discussions/1672

```suggestion
- epic
- faq
```

**Review by aj-stein-nist — DISMISSED** [review]
I remember now, if we can add this before merge, I am happy with this change.

**Review by aj-stein-nist — APPROVED** [review]
Looks good. If I do not receive contrary feedback before end of day 2 March 2023 I will merge this PR and consider it approved.

**Pull request merged**